video_xblock/static/html/studio-edit-default-transcripts.html

```diff
@@ -0,0 +1,115 @@
+    <div class="enabled-default-transcripts-section is-hidden">
+      <div class="default-transcripts-label" value=""></div>
+      <div class="default-transcripts-actions">
+        <a href="#" class="default-transcripts-action-link download-transcript download-setting">
+          {% trans "Download" %}
+        </a>
+        <a href="#"
+           class="default-transcripts-action-link remove-default-transcript"
+           data-lang-code=""
+           data-lang-label=""
+        >
+          {% trans "Delete" %}
+        </a>
+      </div>
+    </div>
+    <!------------------------- End of a hidden block --------------------------->
+
+    <div class="custom-field-section-label available-transcripts">{% trans "Available transcripts" %}</div>
+    {% if default_transcripts %}
+      <!-- Display a list of transcripts fetched from a video platform (API) -->
+        {% for default_transcript in default_transcripts %}
+          <div class="available-default-transcripts-section">
+            <div class="default-transcripts-label" value="{{ default_transcript.lang }}">
+              {{ default_transcript.label }}
+            </div>
+            <div class="default-transcripts-actions">
+              <a class="default-transcripts-action-link upload-default-transcript"
+                 data-change-field-name="{{ field.name }}"
+                 data-lang-code="{{ default_transcript.lang }}"
+                 data-lang-label="{{ default_transcript.label }}"
+                 data-download-url="{{ default_transcript.url }}" >
+                {% trans "Upload" %}
+              </a>
+            </div>
+          </div>
+          <div class="api-request upload-default-transcript {{ default_transcript.lang }} status-error"></div>
+        {% endfor %}
+        {% for transcript in transcripts %}
+          <!-- Success message of enabled transcript removal is to be displayed. -->
+          <div class="api-request remove-default-transcript {{ transcripts.lang }} status-success"></div>
+        {% endfor %}
+        <!----------------------- Hidden block (available sub) ---------------------->
+        <div class="available-default-transcripts-section is-hidden">
+          <div class="default-transcripts-label" value=""></div>
+          <div class="default-transcripts-actions">
+            <a class="default-transcripts-action-link upload-default-transcript"
+               data-change-field-name="{{ field.name }}"
+               data-lang-code=""
+               data-lang-label=""
+               data-download-url="" >
+              {% trans "Upload" %}
+            </a>
+          </div>
+        </div>
+        <!------------------------- End of a hidden block --------------------------->
+    {% else %}
+      <div class="default-transcripts-status status-error">
+        <span class="icon fa fa-remove" aria-hidden="true"></span>
+        {% trans "No timed transcript" %}
+      </div>
+      <div class="tip setting-help">
```

Comment:
There were incorrect html comments near this area, which were confusing syntax highlighter. Fixed.